backend/kesaseteli/applications/views.py

```diff
@@ -163,3 +173,140 @@ def export_and_download_annual_applications(
             return self.render_error(_("Hakemuksia ei löytynyt."))
 
         return self.get_xlsx_response(queryset, columns)
+
+
+class YouthApplicationExcelExportViewSet(AuditLoggingModelViewSet):
+    permission_classes = [AllowAny]  # Permissions are handled per function
+    serializer_class = YouthApplicationExcelExportSerializer
+
+    def create(self, request, *args, **kwargs):
+        return HttpResponse(status=status.HTTP_405_METHOD_NOT_ALLOWED)
+
+    def retrieve(self, request, *args, **kwargs):
+        return HttpResponse(status=status.HTTP_405_METHOD_NOT_ALLOWED)
+
+    def destroy(self, request, *args, **kwargs):
+        return HttpResponse(status=status.HTTP_405_METHOD_NOT_ALLOWED)
+
+    def partial_update(self, request, *args, **kwargs):
+        return HttpResponse(status=status.HTTP_405_METHOD_NOT_ALLOWED)
+
+    def update(self, request, *args, **kwargs):
+        return HttpResponse(status=status.HTTP_405_METHOD_NOT_ALLOWED)
+
+    @classmethod
+    def source_fields_and_output_names(cls) -> typing.OrderedDict[str, str]:
+        with translation.override("fi"):
+            return collections.OrderedDict(
+                [
+                    ("application_date", str(_("Hakupvm"))),
```

Comment:
Probably no point to change as this works perfectly fine also, but just FYI I think in situations like this you could use `gettext` instead of `gettext_lazy` to get rid of those `str(...)`s.

Reply:
Yep, knew that and thought about it a little bit but was a bit unclear how to go about it as gettext_lazy is imported normally as `_` in the codebase, only once gettext is imported as `_`. Simply using gettext directly without renaming could be one option but `gettext` vs. `str(_` + `)` ... hmm it's just 7 vs. 6 characters so its actually quite the same character countwise. Importing gettext as _ somewhere and gettext_lazy as _ somewhere else is would not be a very good idea IMHO as it would make the use of _(...) context specific and thus the code less easily understandable (i.e. having to check which version of gettext has been imported).

Reply:
Tried it out to see what using gettext directly would look like but didn't like the results i.e. they looked less readable to me. So, going with the str(_(...)) although not optimal.